modules/generic_worker/templates/generic-worker.plist.erb

```diff
@@ -27,5 +32,9 @@
             <true/>
         </dict>
     </dict>
+    <key>StandardErrorPath</key>
```

Comment:
+1. This was to get the worker-runner logs in case worker-runner fails. So once run-generic-worker is spawned, this log includes a duplicate of those logs